src/connection.cpp

```diff
@@ -108,21 +105,33 @@ void Connection::accept(Protocol_ptr protocol)
 {
 	this->protocol = protocol;
 	g_dispatcher.addTask(createTask(std::bind(&Protocol::onConnect, protocol)));
+	connectionState = CONNECTION_STATE_CONNECTING_STAGE2;
```

Comment:
Can we get more descriptive names for these stages?

Reply:
```
			CONNECTION_STATE_OPEN,
			CONNECTION_STATE_IDENTIFYING,
			CONNECTION_STATE_READINGS,
			CONNECTION_STATE_CLOSED
```